docs/advanced_usage.rst

```diff
@@ -253,8 +253,7 @@ on the other side, the metadata is retrieved and the trace can continue.
 To propagate the tracing information, HTTP headers are used to transmit the
 required metadata to piece together the trace.
 
-.. autoclass:: ddtrace.propagation.http.HTTPPropagator
-    :members:
+See :py:class:ddtrace.propagation.http.HTTPPropagator for details.
```

Comment:
```suggestion
See :py:class:`ddtrace.propagation.http.HTTPPropagator` for details.
```

Is `:py` needed here?

<img width="546" alt="image" src="https://user-images.githubusercontent.com/1320353/155349854-57c7ff91-75b1-443d-8be1-c77335cfbe50.png">

Reply:
The `:py` is not needed as the default domain is Python. This is currently inconsistent across the docstrings so I will leave this as-is.